src/components/BannedCharacters/BannedCharacters.js

```diff
@@ -0,0 +1,61 @@
+import { h } from 'preact';
+import classNames from 'classnames';
+
+const FONT_SIZE = 18;
+
+const BannedCharacters = ({ bannedCharacters = [], highlightedCharacters = [] }) => (
+    <div class="banned-characters">
+        <span class="label">Banned chars </span>
+        { bannedCharacters.map(char => BannedCharactrer(char, highlightedCharacters.includes(char))) }
+
+        <style jsx>{`
+            .banned-characters {
+                display: inline-flex;
+                align-items: center;
+            }
+
+            .label {
+                font-weight: 600;
+                font-family: sans-serif;
+                font-size: ${FONT_SIZE}px;
+                margin-right: 3px;
+                color: #b3decc;
+                display: inline-block
+            }
+
+            .banned-character {
+                display: inline-block;
+                font-family: monospace;
+                font-weight: 600;
+                font-size: ${FONT_SIZE}px;
+                color: #FFFFFF;
```

Comment:
Maybe it makes sense to move colors to constants alongside with `FONT_SIZE`?

Reply:
You can checkout #18 for reference

Reply:
Updated. But I have a question about file naming. Could you explain cases of usage camel case vs dashes?